src/components/CartItem/CartItem.scss

```diff
@@ -3,140 +3,112 @@
 @import '../../prepare-styles/colors.scss';
 @import '../../prepare-styles/usable-fonts.scss';
 
-
-
-.card-item__container {
-  font-family: 'Mont-SemiBold', sans-serif;
-  border: 1px solid $grayElements;
-  height: 160px;
-  margin-bottom: 16px;
+.product {
+  box-sizing: border-box;
   display: flex;
-  align-items: center;
+  width: 100%;
+  height: 100%;
+  flex-direction: column;
   justify-content: center;
-  background-color: $white;
+  align-items: center;
+  column-gap: 16px;
+  row-gap: 16px;
+  padding: 16px;
+  border: 1px solid $grayElements;
+  height: 160px;
 
   @include on-tablet {
-    margin-bottom: 24px;
+    flex-direction: row;
     height: 128px;
+    padding: 24px;
+    justify-content: space-between;
   }
 
-  @include on-desktop {
-    width: 752px;
-  }
-}
-
-.card-item {
-  margin: 16px;
-  height: auto;
-  width: 100%;
-  display: grid;
-  grid-template-columns: auto auto auto auto;
-  grid-template-rows: auto 50% auto;
-  justify-content: space-between;
-
-  @include on-tablet {
+  &-info {
```

Comment:
it is not a good practice to split classes like `card-item-info` into `.card-item` and `&-info`. i believe it would be better to address them via separate selectors: `.card-item { ... }` and `.card-item-info { ... }`